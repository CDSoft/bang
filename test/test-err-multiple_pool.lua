generator(false)

pool "bar" { depth=12 }
pool "baz" { depth=12 }
pool "bar" { depth=12 } -- same definition
pool "bar" { depth=42 } -- [test/test-err-multiple_pool.lua:5] ERROR: pool bar: multiple definition
