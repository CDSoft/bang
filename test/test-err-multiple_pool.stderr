[test/test-err-multiple_pool.lua:6] ERROR: pool bar: multiple definition
